publisher/src/main/java/com/pavlovmedia/oss/jaxrs/publisher/impl/JaxReference.java

```diff
@@ -10,14 +9,14 @@
  *
  */
 public class JaxReference {
-    public final ServiceReference serivceReference;
+    public final ServiceReference<?> serivceReference;
     public final Object jaxObject;
     
     public Object getJaxObject() {
         return this.jaxObject;
     }
     
-    public JaxReference(final ServiceReference serivceReference, final Object jaxObject) {
+    public JaxReference(final ServiceReference<?> serivceReference, final Object jaxObject) {
```

Comment:
serviceReference mis-spelled again in the original, can we fix it?

Reply:
Yes.